eth_account/_utils/signing.py

```diff
@@ -27,7 +39,11 @@
 STRUCTURED_DATA_SIGN_VERSION = b"\x01"  # Hex value 0x01
 
 
-def sign_transaction_dict(eth_key, transaction_dict, blobs=None):
+def sign_transaction_dict(
+    eth_key: PrivateKey,
+    transaction_dict: Dict[str, Any],
+    blobs: Optional[List[bytes]] = None,
```

Comment:
Maybe it's worth making a blob type, similar to `Hash32`?

Reply:
Done! All `blobs` are now `Blobs` type, imported from a new `types.py`